src/ORM/Table.php

```diff
@@ -243,6 +247,16 @@ public function __construct(array $config = [])
         if (!empty($config['associations'])) {
             $associations = $config['associations'];
         }
+        if (!empty($config['validator'])) {
+            if (!is_array($config['validator'])) {
+                $this->validator(null, $config['validator']);
+            } else {
+                foreach ($config['validator'] as $key => $validator) {
+                    $name = (!is_int($key)) ? $key : null;
```

Comment:
@ADmad but by allowing the key to be skipped wouldn't you avoid the developer having to *know* the internals of the Validator, and the fact that the assumed set is called "default"?